### PR TITLE
src/components: add links to coordinator registration and login

### PR DESCRIPTION
**Pull request by maceksimon**
* Add link to register as coordinator
* Add link to login
* Add translations
* Add tests

### CODE REVIEW

File: src/components/FormRegister.vue [hunk 1]
```diff
@@ -203,6 +209,49 @@ export default defineComponent({
...
+            class="q-mt-none q-mb-md"
+            data-cy="form-register-coordinator-description"
+          >
+            {{ $t('register.form.hintRegisterAsCoordinator') }}
+          </p>
+          <p
+            class="q-mt-md q-mb-none"
+            data-cy="form-register-coordinator-link-wrapper"
+          >
+            <a
```
Comment (tmszi) [inline src/components/FormRegister.vue:232]: Same as my comment bellow https://github.com/auto-mat/ride-to-work-by-bike-frontend/pull/121/files#r1390659229.

If route path /#/register-coordinator doesn't exists yet create some fake with existed layout/component.

File: src/components/FormRegister.vue [hunk 2]
```diff
@@ -203,6 +209,49 @@ export default defineComponent({
...
+              {{ $t('register.form.linkRegisterAsCoordinator') }}
+            </a>
+          </p>
+        </div>
+      </div>
+      <!-- Link: Login -->
+      <div class="q-mt-lg text-body2 text-white" data-cy="form-register-login">
+        <p class="q-my-none">
+          {{ $t('register.form.hintLogin') }}
+          <a
```
[the remark this answers is not in the document]
Comment (maceksimon) [reply src/components/FormRegister.vue]: Added using the `router-link`. Also added router to component test.

File: src/components/FormRegister.vue (hunk 1 above)
Comment (maceksimon) [reply src/components/FormRegister.vue]: Added using the `router-link`. Also added router to component test.
I also fixed this issue in `FormLogin` as the use was identical.
Added route `register-coordinator` (for now, it has the same content as `register`)

**Comment by tmszi** [comment]
Could @maceksimon rebase (doesn't include fix #129) this PR, please?

File: test/cypress/support/component.js [hunk 3]
```diff
@@ -67,6 +71,21 @@ Cypress.Commands.add('mount', (component, options = {}) => {
...
+    options.router = createRouter({
+      routes: routes,
+      history: createMemoryHistory(),
+    });
+  }
+
+  // Add router plugin
+  options.global.plugins.push({
+    install(app) {
+      app.use(options.router);
```
Comment (tmszi) [inline test/cypress/support/component.js:85]: You duplicate code here (check the line 93).  Replace line 97 with line 85  `app.use(options.router);`.

Comment (maceksimon) [reply test/cypress/support/component.js]: I see. Fixed.

File: test/cypress/support/component.js [hunk 4]
```diff
@@ -67,12 +71,21 @@ Cypress.Commands.add('mount', (component, options = {}) => {
...
   // Register Swiper third party lib component
   register();
 
   // Add i18n plugin
   options.global.plugins.push({
     install(app) {
+      app.use(options.router);
       app.use(i18nApp);
       app.use(VueLogger, loggerOptions);
       app.use();
```
Comment (tmszi) [inline test/cypress/support/component.js:91]: We don't need line 91, as I 'm said in my previous comment https://github.com/auto-mat/ride-to-work-by-bike-frontend/pull/121#discussion_r1393646660

Comment (maceksimon) [reply test/cypress/support/component.js]: Ah, I see. Removed.

**Pull request merged**